packages/replay/src/coreHandlers/handleGlobalEvent.ts

```diff
@@ -39,11 +39,13 @@ export function handleGlobalEventListener(replay: ReplayContainer): (event: Even
     }
 
     const exc = event.exception?.values?.[0];
-    addInternalBreadcrumb({
-      message: `Tagging event (${event.event_id}) - ${event.message} - ${exc?.type || 'Unknown'}: ${
-        exc?.value || 'n/a'
-      }`,
-    });
+    if (__DEBUG_BUILD__ && replay.getOptions()._experiments?.traceInternals) {
```

Comment:
`__DEBUG_BUILD__` is turned on by `debug: true` in `init()` right?

Reply:
No, when using `npm install` this is always true I believe (unless you set it in your bundler config yourself), but e.g. in CDNs for "non-debug" bundles this will be false, and thus everything else can be stripped because it is statically an analyzable :)